docs/tutorial_basics.rst

```diff
@@ -184,19 +184,19 @@ To train the dialogue model, run:
 
 .. code-block:: bash
 
-   python -m rasa_core.train -s data/stories.md -d domain.yml -o models/dialogue --epochs 300
```

Comment:
I don't think we should hide these.

Reply:
yeah fair point, i'll just paste the commands from the make file in here